sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Release History
 
-## 1.1.7 (Unreleased)
+## 1.1.7 (2020-09-02)
 
-- Fixed a bug in `BearerTokenAuthenticationPolicy` where a helper class was being instantiated on each request. [PR 10728](https://github.com/Azure/azure-sdk-for-js/pull/10728)
+- `BearerTokenAuthenticationPolicy` now starts trying to refresh the token 30 seconds before the token expires. It will only try to refresh said token on each request when refreshing is not in progress. In the mean time, requests will use the existing token. [PR 10085](https://github.com/Azure/azure-sdk-for-js/pull/10085)
```

Comment:
I don't think we should refer to that PR, since that PR introduced two bugs that were resolved later. Can we leave it without the link?

Reply:
sure, do we have an issue we can point to instead?

Reply:
Yes we have #10084.  Update this PR to point to it instead.

Reply:
That works! thank you.